# -*- coding: utf-8 -*-
# Generated by Django 1.9.7 on 2016-06-21 05:03
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='MapError',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False,
                                        verbose_name='ID')),
                ('created_date', models.DateTimeField(auto_now_add=True)),
                ('solved_date', models.DateTimeField(blank=True, null=True)),
                ('ne_lat', models.FloatField()),
                ('ne_lon', models.FloatField()),
                ('sw_lat', models.FloatField()),
                ('sw_lon', models.FloatField()),
                ('certainty', models.FloatField(blank=True, null=True)),
                ('raster_filename', models.CharField(max_length=512)),
                ('raster_tile_x', models.IntegerField()),
                ('raster_tile_y', models.IntegerField()),
                ('flagged_count', models.IntegerField(default=0)),
                ('state_abbrev', models.CharField(max_length=5)),
            ],
        ),
    ]
